tuf/keys.py

```diff
@@ -86,7 +86,9 @@
 # default.  https://github.com/pyca/ed25519
 _available_crypto_libraries = ['ed25519']
 
-# Import the PyCrypto library so that RSA keys are supported.
+# Try to import TUF's PyCrypto module (pycrypto_keys.py), and verify and record
+# that the PyCrypto library is available.  pycrypto_keys.py is used here for
+# general-purpose cryptography and RSA.
 try:
   import Crypto
```

Comment:
Great comment.

I'd like to note here (for documentation purposes) that integrators are not normally expected to directly import modules like `keys.py`, `pycrypto_keys.py`, etc., but rather the repository tools.

`repository_tool.py` provides functionality that doesn't require PyCrypto nor PyCA's Cryptography, so we instead raise exceptions when a caller tries to use a function that does requires either of these libraries (depending on what is specified in 'tuf.conf').